### DISCUSSION TITLE
JetBrains Mono Regular Nerd Font Complete Mono.ttf is not Monospaced

### DISCUSSION THREAD

**Discussion by daqo98** [ryanoasis/nerd-fonts#760] [Q&A]
Hi everyone!

I'm trying to use it in a zsh terminal in VS Code on Ubuntu 20.04 but the characteristic symbols of oh-my-zsh are not properly depicted (See image). Besides that, there is a lot of spacing. When I open the Font Manager app it says that this font has actually a proportional spacing not monospace. I would appreciate it if someone can correct that.

![WhatsApp Image 2022-01-10 at 20 21 39](https://user-images.githubusercontent.com/68958822/148864754-da5c1208-9215-43c4-806f-fa6ac57d09c8.jpeg)

![WhatsApp Image 2022-01-10 at 20 30 22](https://user-images.githubusercontent.com/68958822/148866457-2b7295ba-7093-46e9-a4c8-1874f73d52f6.jpeg)

On the other hand, the Mono suffix Windows-compatible version works pretty well.

I want to use this field on my VSCode settings.json file across platforms:
`"terminal.integrated.fontFamily": "JetBrainsMono Nerd Font Mono"`

**Comment by Finii** [comment]
I guess the reason are wrong Family Names (#717).

Could you try to remove all `JetBrainsMono Nerd Font` (non-mono) and try again?
It could even by that it interacts with the `JetBrains Mono` (original) fonts :-( Maybe also remove for the test.

My first superficial test show that there are circumstances where it seems to work:
![image](https://user-images.githubusercontent.com/16012374/148875436-4b3935c9-d918-4843-953b-079f56f011dc.png)
